src/main/scala/nl.knaw.dans.easy.bagindex/server/BagIndexServletComponent.scala

```diff
@@ -136,9 +136,8 @@ trait BagIndexServletComponent {
         .flatMap(uuid => databaseAccess.doTransaction(implicit c => database.getBagInfo(uuid)))
         .map(createResponse[BagInfo](bagInfo => <result>{toXml(bagInfo)}</result>)(bagInfo => "result" -> toJson(bagInfo)))
         .recoverWith {
-          case BagIdNotFoundException(_) => Success(createResponse[Unit](_ => <result/>)(_ => "result" -> JArray(List.empty))(()))
+          case BagIdNotFoundException(_) => Success(NotFound(createResponse[Unit](_ => <result/>)(_ => "result" -> JArray(List.empty))(())))
         }
-        .map(Ok(_))
```

Comment:
So where did this `Ok` go to? Shouldn't you add it to the `map` above?

Reply:
It also returns a 200 if we don't explicitly map it to 200. I can move it above it you want. But i works fine like this for all tests (automatic and junit)

Reply:
yes please! I'd prefer explicitness over implicitness here!